src/ILVerify/src/SimpleArrayOfTRuntimeInterfacesAlgorithm.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using Internal.IL;
+using Internal.TypeSystem;
+
+namespace ILVerify
+{
+    internal class SimpleArrayOfTRuntimeInterfacesAlgorithm : RuntimeInterfacesAlgorithm
+    {
+        private DefType[] _arrayRuntimeInterfaces;
+        private MetadataType[] _genericRuntimeInterfaces;
+        private ModuleDesc _systemModule;
+
+        public SimpleArrayOfTRuntimeInterfacesAlgorithm(ModuleDesc systemModule)
+        {
+            _systemModule = systemModule;
+            _arrayRuntimeInterfaces = _systemModule.GetKnownType("System", "Array").RuntimeInterfaces;
+            _genericRuntimeInterfaces = new MetadataType[]
+            {
+                _systemModule.GetKnownType("System.Collections.Generic", "IEnumerable`1"),
+                _systemModule.GetKnownType("System.Collections.Generic", "ICollection`1"),
+                _systemModule.GetKnownType("System.Collections.Generic", "IList`1")
```

Comment:
Yes, I simply overlooked that, thx. According to the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.array?view=netframework-4.7),  `IReadOnlyCollection<T>` is also missing. Maybe also in [Array.CoreRT.cs](https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/src/System/Array.CoreRT.cs#L1539)?